CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### Version 9.3
 * Adds `FallbackFactory`, allowing access to the cause of a Hystrix fallback
+* Adds support for encoded parameters via @Param(encoded = true)
```

Comment:
nit, if you get used to doing backticks, not only will the format look nicer, but you won't accidentally mention someone named Param :P